atomicfu/build.gradle.kts

```diff
@@ -210,7 +210,7 @@ val transformFU by tasks.registering(JavaExec::class) {
     classpath(transformer)
     inputs.dir(classesPreAtomicFuDir)
     outputs.dir(classesPostTransformFU)
-    launcherForJdk(9)
+    launcherForJdk(11)
```

Comment:
Does it make sense to extract launcher JDK version into a constant (or a property) so that the next time we'll need to change it in a single place?

Reply:
Sure, I've pushed a fixup